src/shims/foreign_items.rs

```diff
@@ -160,13 +163,12 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
             // Normally, this will be either `libpanic_unwind` or `libpanic_abort`, but it could
             // also be a custom user-provided implementation via `#![feature(panic_runtime)]`
             "__rust_start_panic" | "__rust_panic_cleanup"=> {
```

Comment:
```suggestion
            "__rust_start_panic" | "__rust_panic_cleanup" => {
```

look, no rustfmt :laughing:

Reply:
FWIW I am just waiting for https://github.com/rust-lang/rustfmt/pull/4090 to finally be available in rustfmt then I'll look into applying it to the codebase and having CI.^^